setup.cfg

```diff
@@ -29,3 +29,20 @@ previous = true
 [flake8]
 max-line-length = 88
 extend-ignore = E203
+
+# mypy related settings
+[mypy]
+plugins = sqlalchemy.ext.mypy.plugin
+[mypy-pyramid.*]
+ignore_missing_imports = True
+[mypy-transaction]
+ignore_missing_imports = True
+[mypy-zope.*]
+ignore_missing_imports = True
+
+[pylint]
+generated-members=
+    scoped_session,
+    Base,
+    DBSession,
+    cls
```

Comment:
this last one, `cls`,  looks a bit dubious to me as it's commonly used on classmethods to reference the class rather than the instance (cls vs self)  so it's name showing up here gives me a bit of a red flag.

Reply:
The line 94(maybe 95) in models.py

 ```    
def __tablename__(cls): # pylint: disable=no-self-argument
        return cls.__name__.lower() 
```
triggers the error "E1101: Instance of 'Base' has no '__name__' member (no-member)", but from my understanding it gets generated when the decoration is done(cls turned from a reference to an instance(a "self") and to the class), Base is already in the list there but that covers to errors later down concerning the metadata member. So I could disable the error for the line, but in my head this was more logical, though not very.

Reply:
For this case, as it's not a truth for "all" cls,  I'd say make it happen on the line.

The other ones ought to be true for all instances by that name, so I'd call those "ok" for this setting.